### PR TITLE
TurboSnap: Detect mismatching entry file and suggest a fix

### PR DESCRIPTION
**Pull request by ghengeveld**

### CODE REVIEW

File: bin-src/lib/getDependentStoryFiles.js [hunk 1]
```diff
@@ -99,7 +99,9 @@ export async function getDependentStoryFiles(ctx, stats, statsPath, changedFiles
   ctx.turboSnap.modules = Object.keys(idsByName);
 
   if (ctx.turboSnap.globs.length === 0) {
-    ctx.log.error(noCSFGlobs({ statsPath, storybookDir, viewLayer }));
+    const entry = stats.modules.find((m) => m.name.endsWith('generated-stories-entry.js'));
```
Comment (tmeasday) [inline bin-src/lib/getDependentStoryFiles.js:102]: This isn't the name of the file in the v7 store. It is called `storybook-stories.js` there: https://github.com/chromaui/chromatic-cli/blob/bf0999da3906fc50f269e6b7573103990c90c765/bin-src/lib/getDependentStoryFiles.js#L70

Comment (ndelangen) [reply bin-src/lib/getDependentStoryFiles.js:102]: Do we need to to support both?

**Review by tmeasday — APPROVED** [review]
Oh, nice!

**Comment by tmeasday** [comment]
@ghengeveld I am surprised there are no Chromatic changes on this PR: https://www.chromatic.com/pullrequest?appId=5d67dc0374b2e300209c41e7&number=490

Has something gone wrong with TS on our own repo?

**Comment by ghengeveld** [comment]
> @ghengeveld I am surprised there are no Chromatic changes on this PR: [chromatic.com/pullrequest?appId=5d67dc0374b2e300209c41e7&number=490](https://www.chromatic.com/pullrequest?appId=5d67dc0374b2e300209c41e7&number=490)
> 
> Has something gone wrong with TS on our own repo?

I think we have a misconfiguration in one of the CI jobs. I'll split them up into separate Chromatic projects because currently a single push triggers 5 or so builds on the same project, which obviously doesn't work well with TurboSnap.

**Pull request merged**